bluemira/equilibria/opt_problems.py

```diff
@@ -815,3 +829,164 @@ def get_state_figure_of_merit(
         sub_opt()
         fom = sub_opt.opt.optimum_value
         return fom
+
+
+class UnconstrainedMinimalErrorCOP(CoilsetOptimisationProblem):
```

Comment:
Ugh, yeah actually this was me not trying to break anything prior to this becoming a feature PR. I will get rid of it. Happy to also generally keep (eq, coilset ..) separate if it might be useful in future.